symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/model/Attachment.java

```diff
@@ -0,0 +1,18 @@
+package com.symphony.bdk.core.service.message.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+import org.apiguardian.api.API;
+
+import java.io.InputStream;
+
+@Getter
+@Setter
+@Accessors(fluent = true)
+@API(status = API.Status.EXPERIMENTAL)
+public class Attachment {
+
+  private InputStream inputStream;
+  private AttachmentType attachmentType;
```

Comment:
don't we need to distinguish the type and the file name?

Reply:
Yes IMHO we should be able to let the user specify a file name.